src/inference/dev_api/openvino/runtime/system_conf.hpp

```diff
@@ -157,6 +186,40 @@ enum ColumnOfProcessorTypeTable {
     PROC_TYPE_TABLE_SIZE = 4   //!< Size of processor type table
 };
 
+/**
+ * @enum       CpuUseStatus
+ * @brief      Defination of CPU_MAP_USED_FLAG column in CPU mapping table.
+ */
+enum CpuUseStatus {
```

Comment:
is it CPU or CPU Processor?
@wangleis what is the better name?

Reply:
Let's use ProcessorUseStatus.